Sasquatch/Sasquatch/ViewControllers/MSDocumentDetailsViewController.swift

```diff
@@ -72,6 +74,32 @@ class MSDocumentDetailsViewController: UIViewController, UITableViewDelegate, UI
     self.presentingViewController?.dismiss(animated: true, completion: nil)
   }
 
+  @IBAction func refreshButtonClicked(_ sender: Any) {
+    var partition = documentContent?.partition ?? ""
+    if (partition.contains(appCenter.kMSDataUserDocumentsPartition)) {
+        partition = appCenter.kMSDataUserDocumentsPartition
+    }
+    self.appCenter.readDocumentWithPartition(partition, documentId: documentContent?.documentId ?? "", documentType: MSDictionaryDocument.self, completionHandler: { (document) in
```

Comment:
I would separate `documentContent?.documentId ?? ""` in a variable too, just for readability

Reply:
Fair point, made the change